design-tokens/src/formats/cssColorModes.ts

```diff
@@ -6,23 +6,62 @@ export const cssColorModes: FormatFn = async ({
   file,
   options,
 }: FormatFnArguments) => {
-  const { outputReferences, theme, mode } = options;
-  const darkTokens = formattedVariables({
+  const { outputReferences, theme, mode: modeArg } = options;
+  const tokens = formattedVariables({
     format: 'css',
     dictionary,
     outputReferences,
     usesDtcg: true,
   });
+
+  // TO DO should default mode be light or "auto" (light aligns with grommet-theme-hpe)
+  const defaultMode = 'light';
```

Comment:
@halocline @britt6612 confirming that default as light mode aligns with your expectations? (this is how tokens currently behaves and how grommet-theme-hpe is defined)

Reply:
I would agree with this.

Reply:
I'm fine with that. Can revisit in the future if needed.